src/Microsoft.DotNet.Darc/src/DarcLib/Models/Darc/DependencyGraph.cs

```diff
@@ -92,22 +138,23 @@ public class DependencyGraph
         /// <param name="repoUri">Root repository URI</param>
         /// <param name="commit">Root commit</param>
         /// <param name="includeToolset">If true, toolset dependencies are included.</param>
+        /// <param name="lookupBuilds">If true, the builds contributing to each node are looked up. Must be a remote build.</param>
```

Comment:
>ookupB [](start = 26, length = 6)

Hrm... this comment doesn't match the actual change.  This was probably when there was only 1 "option" that was a boolean, and then you packed them up into an object and this got left behind. :-)